hanami_application/slices/crm/actions/customers/create.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'securerandom'
+
+module Crm
+  module Actions
+    module Customers
+      class Create < Action::Base
+        include Deps['application.command_bus', 'repositories.customers']
+
+        def handle(req, res)
+          params = JSON.parse(req.body.read, symbolize_names: true)
+          create_customer(params[:customer_id], params[:name])
+        rescue Crm::Customer::AlreadyRegistered
+          flash[:notice] = "Customer was already registered"
+          render "new"
```

Comment:
```suggestion
          res.flash[:notice] = "Customer was already registered"
          render "new"
```

But does it make sense to set `flash` when it's never read?

Also, `render "new"` does not work, but since this is API-only for now, shouldn't it be something like:

```
          res.body = { error: "Customer was already registered" }
```
?